.github/workflows/pr-labeler.yml

```diff
@@ -25,13 +25,15 @@ jobs:
         with:
           repo-token: ${{ secrets.PR_TOKEN }} 
           remove-labels: "safe to test"
+          remove-labels: "viam-employee"
       - name: Auto label org members
         if: |
           steps.is_organization_member.outputs.result == 'true'
         uses: andymckay/labeler@1.0.4
         with:
           repo-token: ${{ secrets.PR_TOKEN }} 
           add-labels: "safe to test"
+          add-labels: "viam-employee"
```

Comment:
I would name it viam-org-member. Not all members are necessarily employees

Reply:
Good point. Updated.